nwplus/nwhacks2019
# Web/fix/feature flag typo

:tickets: **Ticket(s)**: Closes #

---

## :construction_worker: Changes

This fixes a feature flag typo in the volunteer application. Previously it checked that the hacker application feature flag was enabled instead of the volunteer application feature flag.

## :flashlight: Testing Instructions

Already tested, but if you want to double check that it works just:
1. `make web` 
2. Toggle the `enabled` field in the `feature_flags/volunteer_application` through the firebase console.
3. Check that the volunteer application is enabled/disabled successfully.
